src/main/java/net/dv8tion/jda/api/interactions/commands/build/OptionData.java

```diff
@@ -226,6 +262,41 @@ public OptionData setRequired(boolean required)
         return this;
     }
 
+    /**
+     * Add a predefined choice for this option.
+     * <br>The user can only provide one of the choices and cannot specify any other value.
+     * 
+     * @param  name
+     *         The name used in the client, up to {@value #MAX_NAME_LENGTH} characters long, as defined by
+     *         {@link #MAX_NAME_LENGTH}
+     * @param  value
+     *         The value received in {@link net.dv8tion.jda.api.interactions.commands.OptionMapping OptionMapping}
+     * 
+     * @throws IllegalArgumentException
+     *         If any of the following checks fail
+     *         <ul>
+     *             <li>{@code name} is not null, empty and less or equal to {@value #MAX_NAME_LENGTH} characters long</li>
+     *             <li>{@code value} is not less than {@link #MIN_NEGATIVE_NUMBER} and not larger than {@link #MAX_POSITIVE_NUMBER}</li>
+     *             <li>The amount of already set choices is less than {@link #MAX_CHOICES}</li>
+     *             <li>The {@link OptionType} is {@link OptionType#NUMBER}</li>
+     *         </ul>
+     * 
+     * @return The OptionData instance, for chaining
+     */
+    @Nonnull
+    public OptionData addChoice(@Nonnull String name, double value)
+    {
+        Checks.notEmpty(name, "Name");
+        Checks.notLonger(name, MAX_NAME_LENGTH, "Name");
+        Checks.check(value >= MIN_NEGATIVE_NUMBER, "Double value may not be lower than %f", MIN_NEGATIVE_NUMBER);
+        Checks.check(value <= MAX_POSITIVE_NUMBER, "Double value may not be larger than %f", MAX_POSITIVE_NUMBER);
```

Comment:
Refer to earlier comments; just putting this here, as it may be easier to change these checks than change the definitions.  I'm not sure which is better for readability, too.

Reply:
Copying previous statement: The constant should represent the largest _valid_ value.